src/Accordion.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { createBootstrapComponent } from './ThemeProvider';
+import Button from './Button';
+import Card from './Card';
+import Collapse from './Collapse';
+
+class Accordion extends Component {
+  static propTypes = {
+    children: PropTypes.instanceOf(Object),
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      openedId: 0,
+    };
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick = id => {
+    this.setState({
+      openedId: id,
+    });
+  };
+
+  render() {
+    const { children } = this.props;
+    return (
+      <div>
+        {React.Children.map(children, (CurCard, i) => (
```

Comment:
I'm not sure about this approach. Perhaps using some sort of AccordionContext might be better?

Reply:
This is where I ultimately got stuck. Any suggestion for how I should implement `AccordionContext`? I might take a look at how other `...Context` components were implemented

Reply:
i think the idea is to check for the context inside the card header, and interact with it if present

we use similar patterns around navs and tabs

Reply:
@bpas247 yeah I think the API might be something like 

```jsx
<Accordion>
  <Card>
	<Card.Header>
	  <Accordion.Toggle eventKey="1">
		<Button >Click me</Button>
	  </Accordion.Toggle>
 	</Card.Header>
	<Collapse eventKey="1">
		{/* body etc blah blah */}
	</Collapse>
  </Card>
</Accordion>
```

It's verbose but I think probably the right starting point as a raw API

Reply:
I'll go with that approach @jquense :+1: I'm hoping I can get that working by this weekend